### PR TITLE
time picker minHour, maxHour, minMinute and maxMinute properties added

### LINKED ISSUE #1109: min/max and intervals values on date / time pickers
we need min / max and intervals on the native date / time pickers....

### PR DESCRIPTION
**Pull request by enchev**
Partially resolved #1109

### CODE REVIEW

**Comment by enchev** [comment]
run ci

**Comment by enchev** [comment]
run ci

**Comment by hamorphis** [comment]
:+1:

**Comment by nsndeck** [comment]
:+1:

**Comment by ErjanGavalji** [comment]
:+1:

File: ui/time-picker/time-picker-common.ts [hunk 1]
```diff
@@ -2,10 +2,62 @@
 import dependencyObservable = require("ui/core/dependency-observable");
 import proxy = require("ui/core/proxy");
 import view = require("ui/core/view");
+import types = require("utils/types");
+
+function isHourValid(value: number): boolean {
+    return types.isNumber(value) && value >= 1 && value <= 23;
```
Comment (vakrilov) [inline ui/time-picker/time-picker-common.ts]: Isn't 0 a valid hour - probably the check should be `value >= 0`

**Pull request merged**